lib/beaker/host_prebuilt_steps.rb

```diff
@@ -360,6 +360,10 @@ def copy_ssh_to_root host, opts
         else
           host.exec(Command.new('sudo su -c "cp -r .ssh /root/."'), {:pty => true})
         end
+
+        if host.exec(Command.new('sudo selinuxenabled'), :accept_all_exit_codes => true).exit_code == 0
```

Comment:
I haven't had time to get to this. If someone wants to just get it in correctly and abandon my PR, that's fine. Otherwise, It might not be too soon.